QRCodeHere/ContentView.swift

```diff
@@ -51,6 +53,7 @@ struct ContentView: View {
                                 .resizable()
                                 .scaledToFit()
                                 .frame(width: 280, height: 280, alignment: .center)
+                                .watermarked(with: watermark.isEmpty ? nil : watermark)
```

Comment:
This along with my suggestions in the `Watermark` view modifier will make the layout better and prevent pushing views above and bellow the image.

Instead of the hardcoded 256 value we could calculate the height of the watermark text using `NSFont.preferredFont(forTextStyle:options:)` and subtract it from 280, but that would require us to change the minimum deployment target to macOS 11. The code to calculate the height would be:
```swift
var watermarkHeight: CGFloat {
    let attributes = [NSAttributedString.Key.font: NSFont.prefferedFont(forTextStyle: .caption1, options: [:])]
    return watermark.size(withAttributes: attributes).height
}
```

```suggestion
                                .frame(width: watermark.isEmpty ? 280 : 256, height: watermark.isEmpty ? 280 : 256)
                                .watermarked(with: watermark.isEmpty ? nil : watermark)
                                .frame(maxHeight: 280)
```

Reply:
I added the suggested change!

Nice idea btw 😍

For the font calculation, maybe in later phase as currently it's better to focus on the core feature of this app which is giving the user a fast way to share content in mails or to his iPhone directly in safe app (open source)